src/entities/socialProfile.ts

```diff
@@ -66,6 +68,10 @@ export class SocialProfile extends BaseEntity {
   @Column('text', { nullable: true })
   socialNetworkId: string;
```

Comment:
Add an index for socialNetworkId and socialNetwork which are queried in ```findSocialProfileBySocialNetworkIdAndSocialNetwork```

```@Index(["socialNetworkId", "socialNetwork"])```

Reply:
Great